### ISSUE TITLE
Change cache location from ~/.mapscii to ~/.config/.mapscii

### ISSUE THREAD

**Issue by Leosxe** [rastapasta/mapscii#60]
As suggested in this video (https://www.youtube.com/watch?v=AFtfpluqv14&t=585s) is a good practice to use the default folder, that is ~/.config to put application files to have a cleaner home folder, so I would suggest updating the code and move the .mapscii folder in that folder.

**Comment by jaller94** [comment]
Sounds like a great idea to allow the system or an administrator to clean up the cache or mount the `~/.cache` to a temporary storage. 👍

Actual configs should go somewhere else (if mapscii ever gets those).

I think it should be `~/.cache/mapscii` instead of `~/.cache/.mapscii`. Is there any reason to make the sub folder a hidden folder?

**Comment by Leosxe** [comment]
Yeah, I think that <code>~/.cache/mapscii</code> is better, there is no reason to make a hidden folder.

**Comment by jaller94** [comment]
Should a terminal message suggest users to clean up `~/.mapscii/cache`? Should the files be copied over?

**Comment by PittacusOfLorien** [comment]
Is this going to be implemented sometime soon?
